docs/Encyclopedia/pages/TCGA_VCF_1.1v2.md

```diff
@@ -107,7 +107,7 @@ sample-specific metadata.
 
 All TCGA-specific additions and modifications in [validation
 steps](#validation-rules) are prefixed with a
-<TCGA-VCF> tag for convenient comparison with 1000Genomes VCF 4.1. The
+<TCGA-VCF\> tag for convenient comparison with 1000Genomes VCF 4.1. The

```

Comment:
Is there any documentation that states which characters need to be escaped?

Reply:
This seems like a pretty good reference: https://github.com/mattcone/markdown-guide/blob/master/_basic-syntax/escaping-characters.md#characters-you-can-escape